ui/lib/pki/addon/routes.js

```diff
@@ -3,6 +3,7 @@ import buildRoutes from 'ember-engines/routes';
 export default buildRoutes(function () {
   this.route('overview');
   this.route('configuration', function () {
+    this.route('index', { path: '/' });
```

Comment:
Added this route so that the "read configuration" contents don't render on child paths